doc/man/zcashd.1

```diff
@@ -382,7 +382,7 @@ Generate coins (default: 0)
 .HP
 \fB\-genproclimit=\fR<n>
 .IP
-Set the number of threads for coin generation if enabled (\fB\-1\fR = all
+Set the number of threads for coin generation if enabled (\fB\fR = all
```

Comment:
I don't know and suspect it is incorrect. I don't know yet how to verify this, hang on…